src/views/virtualmachines/details/tabs/configuration/storage/components/tables/disk/DiskRow.tsx

```diff
@@ -72,15 +82,17 @@ const DiskRow: FC<
       </TableData>
 
       <TableData activeColumnIDs={activeColumnIDs} id="source">
-        {isPVCSource(obj) ? (
+        {sourcesLoaded && hasPVC && (
```

Comment:
Is sourcesLoaded needed because it affects the hasPVC const?

Reply:
Why does it affect the const?